testsuite/running-tests.md

```diff
@@ -5,55 +5,33 @@ This document will enumerate everything you need to do run the sonobuoy based co
 ## Prerequisites
 
 1. Install [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl).
-1. Set the `KUBECONFIG` environment variable to the path to your kubeconfig file of your cluster.
-2. Install [sonobuoy](https://github.com/vmware-tanzu/sonobuoy#installation) version 0.55.1 or higher. Run `sonobuoy version` to verify it's installed correctly.
+2. Set the `KUBECONFIG` environment variable to the path to your kubeconfig file of your cluster.
 3. Address the [network requirements](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/quickstart-connect-cluster#meet-network-requirements) on your cluster for the Arc agents to communicate with Azure.
-4. Download and install [git](https://git-scm.com/downloads).
-5. Download and install [az cli](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
-6. Download and install [python 3](https://www.python.org/downloads)
 
-### Additional Prerequisites for Arc enabled Data Services
-
-1. Install [arcdata extension](https://docs.microsoft.com/en-us/azure/azure-arc/data/release-notes).
 
 ## Running the script and publishing the results
 
-1. Clone this repository.
-2. Edit the [`partner-metadata.md`](partner-metadata.md) file and fill in the required details. For reference, please see the [`partner-metadata-sample.md`](partner-metadata-sample.md) file.
-
-### Arc enabled Kubernetes
-1. Edit the [`k8s-conformance-test-suite.sh`](k8s-conformance-test-suite.sh) file and set the values for the required environment variables.
-2. If your cluster is behind an outbound proxy, please edit the above file according to the instruction provided as comments for proxy configuration.
-3. Make the test suite file executable by running `chmod +x k8s-conformance-test-suite.sh`.
-4. Execute the script by running `./k8s-conformance-test-suite.sh`.
-5. The test suite will take the storage account details as environment variables and will handle publishing the results in the right format.
-
-*The above script is **bash only**. Please use the [`k8s-conformance-test-suite.ps1`](k8s-conformance-test-suite.ps1) script for windows hosts.
+1. Clone three files azure-arc-conformance.properties, launch.yaml and kustomization.yaml.
+2. Create a directory i.e., conformanceTest and move azure-arc-conformance.properties, launch.yaml and kustomization.yaml to the directory
```

Comment:
We won't need this right? The repo itself maintains these in the [same directory](https://github.com/santosh02iiit/azure-arc-validation/tree/launcher_VMwareProposal/testsuite). They just need to run `kubectl apply` from here.

Edit this section as follows:

### Arc enabled Kubernetes
1. After cloning the repo, navigate to the testsuite directory from the repo root: `cd testsuite`.
2. Edit the [azure-arc-conformance.properties](azure-arc-conformance.properties) file and fill in the required environment variables. You will be provided the credentials to do so.
3. Run the commands as follows: `kubectl apply -f kustomize.yaml`, `kubectl apply -f launch.yaml`.
4. The test suite will take care of publishing the results to the storage account.